graphql-kotlin-spring-server/src/test/kotlin/com/expediagroup/graphql/spring/routes/SubscriptionRoutesConfigurationIT.kt

```diff
@@ -130,6 +130,8 @@ class SubscriptionRoutesConfigurationIT(
             .verify()
     }
 
+    private fun convertValueToString(message: SubscriptionOperationMessage) = objectMapper.writeValueAsString(message)
```

Comment:
unsure how useful this method is -> it is just a test class anyway....

Reply:
Will delete